generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -157,17 +157,11 @@ export class <%= entityAngularName %>DialogComponent implements OnInit {
         this.activeModal.dismiss(result);
     }
 
-    private onSaveError(error) {
-        try {
-            error.json();
-        } catch (exception) {
-            error.message = error.text();
-        }
+    private onSaveError() {
         this.isSaving = false;
-        this.onError(error);
```

Comment:
Are you seeing error message in the screen after removing this? I need to find time to test this